modules/develop/pages/data-transforms.adoc

```diff
@@ -0,0 +1,253 @@
+= Data Transforms
+:description: Learn how to run WebAssembly data transforms within Redpanda brokers.
+:page-aliases: reference:rpk/rpk-wasm/rpk-wasm.adoc, reference:rpk/rpk-wasm.adoc, reference:rpk/rpk-wasm/rpk-wasm-deploy.adoc, reference:rpk/rpk-wasm/rpk-wasm-generate.adoc, reference:rpk/rpk-wasm/rpk-wasm-remove.adoc, data-management:data-transform.adoc
+:latest-data-transforms-version: 0.0.0-20230830git604fcce
+
+Redpanda provides the framework to build and deploy inline transformations on data written to Redpanda topics, delivering processed and validated data to consumers in the format they expect. Data transforms let you run common data streaming tasks, like filtering, scrubbing, and transcoding, within the Redpanda broker. For example, you may have consumers that require you to remove credit card numbers or convert JSON to Avro. Data transforms can also interact with the Redpanda Schema Registry to work with encoded data types. Redpanda does this directly inside the broker, eliminating the need to manage a separate stream processing environment or use 3rd-party tools. 
+
+image::shared:wasm1.png[Data transforms in a broker] 
+
+Data transforms use a WebAssembly (Wasm) engine inside a Redpanda broker. They run asynchronously from the source topic to the destination topic, which allows Redpanda to control the entire transform lifecycle. For example, Redpanda can stop and start transforms when partitions are moved or to free up system resources for other tasks. You can develop and manage data transforms with `rpk transform` commands.
```

Comment:
> asynchronously

We do cover this in the system sequence diagram below really well, I wonder if we should introduce the async concept then (or just not mention it here?)

Reply:
moved it down closer to that sequence diagram!